### ISSUE TITLE
SEACAS CGNS tests failing in ATDM Trilinos cts1 intel-18.0.2 openmpi-2.0.3 builds starting 2020-01-03

### ISSUE THREAD

**Issue by bartlettroscoe** [trilinos/Trilinos#6540]
CC: @trilinos/seacas , @kddevin  (Trilinos Data Services Product Lead), @bartlettroscoe

<Checklist>
<???: Add label "client: ATDM">
<???: Add label "ATDM Sev: Blocker" (by default but could be other "ATDM Sev: XXX")>
<???: Add label "type: bug"?>
<???: Add label for affected packages (e.g. "pkg: MueLu", "pkg: Tpetra", "pkg: Kokkos", etc.)>
<???: Add label "PA: ???Project Area???" (e.g. "PA: Linear Solvers", "PA: Data Services")>
<???: Add milestone "Initial cleanup of new ATDM ..." or "Keep promoted ATDM ...">
<???: Once GitHub Issue is created, add entries for tests to TrilinosATDMStatus/*.csv files>

## Next Action Status

<status-and-or-first-action>

## Description

As shown in [this query](https://testing-dev.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-01-01&end=2020-01-06&filtercount=4&showfilters=1&filtercombine=and&field1=buildname&compare1=65&value1=Trilinos-atdm-cts1&field2=groupname&compare2=62&value2=Experimental&field3=testname&compare3=65&value3=SEACAS&field4=status&compare4=61&value4=failed) the tests:

* `SEACASIoss_structured_cgns_append`
* `SEACASIoss_structured_cgns_roundtrip_reordered_zones_fields`
* `SEACASIoss_structured_cgns_roundtrip_with_flush`

are failing in the new ATDM Trilinos 'cts1' builds (see [ATDV-212](https://sems-atlassian-srn.sandia.gov/browse/ATDV-280)):

in the builds:

* `Trilinos-atdm-cts1-intel-18.0.2_openmpi-2.0.3_openmp_static_dbg`
* `Trilinos-atdm-cts1-intel-18.0.2_openmpi-2.0.3_openmp_static_opt`

which first got set up and running on CDash on 2020-01-03.

## Current Status on CDash

* [SEACAS tests in `Trilinos-atdm-cts1-intel-18.0.2_openmpi-2.0.3_openmp_static*` builds for current testing day](https://testing-dev.sandia.gov/cdash/queryTests.php?project=Trilinos&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=65&value1=Trilinos-atdm-cts1-intel-18.0.2_openmpi-2.0.3_openmp_static&field2=groupname&compare2=62&value2=Experimental&field3=testname&compare3=65&value3=SEACAS)

## Steps to Reproduce

One should be able to reproduce this failure on any of the cts1 machines as described in:

* https://github.com/trilinos/Trilinos/blob/develop/cmake/std/atdm/README.md

More specifically, the commands given for the system <supported-atdm-system> are provided at:

* https://github.com/trilinos/Trilinos/blob/develop/cmake/std/atdm/README.md#tlcc-2-and-cts-1

The exact commands to reproduce this issue should be:

```
$ cd <some_build_dir>/

$ source $TRILINOS_DIR/cmake/std/atdm/load-env.sh \
    Trilinos-atdm-cts1-intel-18.0.2_openmpi-2.0.3_openmp_static_opt

$ cmake \
 -GNinja \
 -DTrilinos_CONFIGURE_OPTIONS_FILE:STRING=cmake/std/atdm/ATDMDevEnv.cmake \
 -DTrilinos_ENABLE_TESTS=ON -DTrilinos_ENABLE_SEACAS=ON \
 $TRILINOS_DIR

$ make NP=4

$ salloc -N1 --time=0:20:00 --account=<YOUR_WCID>  ctest -j4
```

**Comment by bartlettroscoe** [comment]
FYI: As shown in [this query](https://testing-dev.sandia.gov/cdash/testSummary.php?project=1&name=SEACASIoss_structured_cgns_append&date=2020-01-06), these tests are passing okay in the other new 'cts1' builds:

* `Trilinos-atdm-cts1-intel-19.0.5_openmpi-4.0.1_openmp_static_dbg`
* `Trilinos-atdm-cts1-intel-19.0.5_openmpi-4.0.1_openmp_static_opt`

as well as the 'cee-rhel6' builds.

Since SPARC will be dropping the builds:

* `Trilinos-atdm-cts1-intel-18.0.2_openmpi-2.0.3_openmp_static_dbg`
* `Trilinos-atdm-cts1-intel-18.0.2_openmpi-2.0.3_openmp_static_opt`

soon and since these tests pass in the `cts1-intel-19.0.5_openmpi-4.0.1` builds, I would not bother trying to triage and fix this (unless there is some desire to do so).

Therefore, I will disable these failing tests in the builds:

* `Trilinos-atdm-cts1-intel-18.0.2_openmpi-2.0.3_openmp_static_dbg`
* `Trilinos-atdm-cts1-intel-18.0.2_openmpi-2.0.3_openmp_static_opt`

and then someone can mark this issue with "wontfix" and we can close this (but not until these tests can be disabled).

**Comment by grover-trilinos** [comment]
<html>

<body>

<h2>Test results for issue #6540 as of 2020-08-16</h2>

<p>
<font color="green">Tests with issue trackers Passed: twip=6</font><br>
</p>

<details>

<summary><b>Detailed test results:</b> (click to expand)</b></summary>

<h3><font color="green">Tests with issue trackers Passed: twip=6</font></h3>
<table style="width:100%" boarder="1">

<tr>
<th>Site</th>
<th>Build Name</th>
<th>Test Name</th>
<th>Status</th>
<th>Details</th>
<th>Consec&shy;utive Pass Days</th>
<th>Non-pass Last 30 Days</th>
<th>Pass Last 30 Days</th>
<th>Issue Tracker</th>
</tr>

<tr>
<td align="left">eclipse</td>
<td align="left"><a href="https://testing.sandia.gov/cdash/index.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=2&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=site&compare2=61&value2=eclipse">Trilinos-atdm-cts1empire-intel-18.0.2_&shy;openmpi-4.0.1_&shy;openmp_&shy;static_&shy;dbg</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/25949240">SEACASIoss_&shy;structured_&shy;cgns_&shy;append</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/25949240"><font color="green">Passed</font></a></td>
<td align="left">Completed</td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_append&field3=site&compare3=61&value3=eclipse"><font color="green">29</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_append&field3=site&compare3=61&value3=eclipse"><font color="red">0</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_append&field3=site&compare3=61&value3=eclipse"><font color="green">29</font></a></td>
<td align="right"><a href="https://github.com/trilinos/Trilinos/issues/6540">#6540</a></td>
</tr>

<tr>
<td align="left">eclipse</td>
<td align="left"><a href="https://testing.sandia.gov/cdash/index.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=2&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=site&compare2=61&value2=eclipse">Trilinos-atdm-cts1empire-intel-18.0.2_&shy;openmpi-4.0.1_&shy;openmp_&shy;static_&shy;opt</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/25942469">SEACASIoss_&shy;structured_&shy;cgns_&shy;append</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/25942469"><font color="green">Passed</font></a></td>
<td align="left">Completed</td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_append&field3=site&compare3=61&value3=eclipse"><font color="green">30</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_append&field3=site&compare3=61&value3=eclipse"><font color="red">0</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_append&field3=site&compare3=61&value3=eclipse"><font color="green">30</font></a></td>
<td align="right"><a href="https://github.com/trilinos/Trilinos/issues/6540">#6540</a></td>
</tr>

<tr>
<td align="left">eclipse</td>
<td align="left"><a href="https://testing.sandia.gov/cdash/index.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=2&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=site&compare2=61&value2=eclipse">Trilinos-atdm-cts1empire-intel-18.0.2_&shy;openmpi-4.0.1_&shy;openmp_&shy;static_&shy;dbg</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/25949265">SEACASIoss_&shy;structured_&shy;cgns_&shy;roundtrip_&shy;reordered_&shy;zones_&shy;fields</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/25949265"><font color="green">Passed</font></a></td>
<td align="left">Completed</td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_reordered_zones_fields&field3=site&compare3=61&value3=eclipse"><font color="green">29</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_reordered_zones_fields&field3=site&compare3=61&value3=eclipse"><font color="red">0</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_reordered_zones_fields&field3=site&compare3=61&value3=eclipse"><font color="green">29</font></a></td>
<td align="right"><a href="https://github.com/trilinos/Trilinos/issues/6540">#6540</a></td>
</tr>

<tr>
<td align="left">eclipse</td>
<td align="left"><a href="https://testing.sandia.gov/cdash/index.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=2&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=site&compare2=61&value2=eclipse">Trilinos-atdm-cts1empire-intel-18.0.2_&shy;openmpi-4.0.1_&shy;openmp_&shy;static_&shy;opt</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/25942483">SEACASIoss_&shy;structured_&shy;cgns_&shy;roundtrip_&shy;reordered_&shy;zones_&shy;fields</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/25942483"><font color="green">Passed</font></a></td>
<td align="left">Completed</td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_reordered_zones_fields&field3=site&compare3=61&value3=eclipse"><font color="green">30</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_reordered_zones_fields&field3=site&compare3=61&value3=eclipse"><font color="red">0</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_reordered_zones_fields&field3=site&compare3=61&value3=eclipse"><font color="green">30</font></a></td>
<td align="right"><a href="https://github.com/trilinos/Trilinos/issues/6540">#6540</a></td>
</tr>

<tr>
<td align="left">eclipse</td>
<td align="left"><a href="https://testing.sandia.gov/cdash/index.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=2&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=site&compare2=61&value2=eclipse">Trilinos-atdm-cts1empire-intel-18.0.2_&shy;openmpi-4.0.1_&shy;openmp_&shy;static_&shy;dbg</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/25949276">SEACASIoss_&shy;structured_&shy;cgns_&shy;roundtrip_&shy;with_&shy;flush</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/25949276"><font color="green">Passed</font></a></td>
<td align="left">Completed</td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_with_flush&field3=site&compare3=61&value3=eclipse"><font color="green">29</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_with_flush&field3=site&compare3=61&value3=eclipse"><font color="red">0</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_with_flush&field3=site&compare3=61&value3=eclipse"><font color="green">29</font></a></td>
<td align="right"><a href="https://github.com/trilinos/Trilinos/issues/6540">#6540</a></td>
</tr>

<tr>
<td align="left">eclipse</td>
<td align="left"><a href="https://testing.sandia.gov/cdash/index.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=2&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=site&compare2=61&value2=eclipse">Trilinos-atdm-cts1empire-intel-18.0.2_&shy;openmpi-4.0.1_&shy;openmp_&shy;static_&shy;opt</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/25942669">SEACASIoss_&shy;structured_&shy;cgns_&shy;roundtrip_&shy;with_&shy;flush</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/25942669"><font color="green">Passed</font></a></td>
<td align="left">Completed</td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_with_flush&field3=site&compare3=61&value3=eclipse"><font color="green">30</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_with_flush&field3=site&compare3=61&value3=eclipse"><font color="red">0</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-18&end=2020-08-16&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_with_flush&field3=site&compare3=61&value3=eclipse"><font color="green">30</font></a></td>
<td align="right"><a href="https://github.com/trilinos/Trilinos/issues/6540">#6540</a></td>
</tr>

</table>


</details>

</body>

</html>

This is an automated comment generated by Grover. Each week, Grover collates and reports data from CDash in an automated way to make it easier for developers to stay on top of their issues. Grover saw that there tests being tracked on CDash that are associated with this open issue. If you have a question, please reach out to Ross. I'm just a cat.

**Comment by grover-trilinos** [comment]
<html>

<body>

<h2>Test results for issue #6540 as of 2020-08-23</h2>

<p>
<font color="green">Tests with issue trackers Passed: twip=6</font><br>
</p>

<details>

<summary><b>Detailed test results:</b> (click to expand)</b></summary>

<h3><font color="green">Tests with issue trackers Passed: twip=6</font></h3>
<table style="width:100%" boarder="1">

<tr>
<th>Site</th>
<th>Build Name</th>
<th>Test Name</th>
<th>Status</th>
<th>Details</th>
<th>Consec&shy;utive Pass Days</th>
<th>Non-pass Last 30 Days</th>
<th>Pass Last 30 Days</th>
<th>Issue Tracker</th>
</tr>

<tr>
<td align="left">eclipse</td>
<td align="left"><a href="https://testing.sandia.gov/cdash/index.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=2&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=site&compare2=61&value2=eclipse">Trilinos-atdm-cts1empire-intel-18.0.2_&shy;openmpi-4.0.1_&shy;openmp_&shy;static_&shy;dbg</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/27167305">SEACASIoss_&shy;structured_&shy;cgns_&shy;append</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/27167305"><font color="green">Passed</font></a></td>
<td align="left">Completed</td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_append&field3=site&compare3=61&value3=eclipse"><font color="green">28</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_append&field3=site&compare3=61&value3=eclipse"><font color="red">0</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_append&field3=site&compare3=61&value3=eclipse"><font color="green">28</font></a></td>
<td align="right"><a href="https://github.com/trilinos/Trilinos/issues/6540">#6540</a></td>
</tr>

<tr>
<td align="left">eclipse</td>
<td align="left"><a href="https://testing.sandia.gov/cdash/index.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=2&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=site&compare2=61&value2=eclipse">Trilinos-atdm-cts1empire-intel-18.0.2_&shy;openmpi-4.0.1_&shy;openmp_&shy;static_&shy;opt</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/27154102">SEACASIoss_&shy;structured_&shy;cgns_&shy;append</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/27154102"><font color="green">Passed</font></a></td>
<td align="left">Completed</td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_append&field3=site&compare3=61&value3=eclipse"><font color="green">29</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_append&field3=site&compare3=61&value3=eclipse"><font color="red">0</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_append&field3=site&compare3=61&value3=eclipse"><font color="green">29</font></a></td>
<td align="right"><a href="https://github.com/trilinos/Trilinos/issues/6540">#6540</a></td>
</tr>

<tr>
<td align="left">eclipse</td>
<td align="left"><a href="https://testing.sandia.gov/cdash/index.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=2&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=site&compare2=61&value2=eclipse">Trilinos-atdm-cts1empire-intel-18.0.2_&shy;openmpi-4.0.1_&shy;openmp_&shy;static_&shy;dbg</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/27167329">SEACASIoss_&shy;structured_&shy;cgns_&shy;roundtrip_&shy;reordered_&shy;zones_&shy;fields</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/27167329"><font color="green">Passed</font></a></td>
<td align="left">Completed</td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_reordered_zones_fields&field3=site&compare3=61&value3=eclipse"><font color="green">28</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_reordered_zones_fields&field3=site&compare3=61&value3=eclipse"><font color="red">0</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_reordered_zones_fields&field3=site&compare3=61&value3=eclipse"><font color="green">28</font></a></td>
<td align="right"><a href="https://github.com/trilinos/Trilinos/issues/6540">#6540</a></td>
</tr>

<tr>
<td align="left">eclipse</td>
<td align="left"><a href="https://testing.sandia.gov/cdash/index.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=2&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=site&compare2=61&value2=eclipse">Trilinos-atdm-cts1empire-intel-18.0.2_&shy;openmpi-4.0.1_&shy;openmp_&shy;static_&shy;opt</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/27154110">SEACASIoss_&shy;structured_&shy;cgns_&shy;roundtrip_&shy;reordered_&shy;zones_&shy;fields</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/27154110"><font color="green">Passed</font></a></td>
<td align="left">Completed</td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_reordered_zones_fields&field3=site&compare3=61&value3=eclipse"><font color="green">29</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_reordered_zones_fields&field3=site&compare3=61&value3=eclipse"><font color="red">0</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_reordered_zones_fields&field3=site&compare3=61&value3=eclipse"><font color="green">29</font></a></td>
<td align="right"><a href="https://github.com/trilinos/Trilinos/issues/6540">#6540</a></td>
</tr>

<tr>
<td align="left">eclipse</td>
<td align="left"><a href="https://testing.sandia.gov/cdash/index.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=2&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=site&compare2=61&value2=eclipse">Trilinos-atdm-cts1empire-intel-18.0.2_&shy;openmpi-4.0.1_&shy;openmp_&shy;static_&shy;dbg</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/27167349">SEACASIoss_&shy;structured_&shy;cgns_&shy;roundtrip_&shy;with_&shy;flush</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/27167349"><font color="green">Passed</font></a></td>
<td align="left">Completed</td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_with_flush&field3=site&compare3=61&value3=eclipse"><font color="green">28</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_with_flush&field3=site&compare3=61&value3=eclipse"><font color="red">0</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_with_flush&field3=site&compare3=61&value3=eclipse"><font color="green">28</font></a></td>
<td align="right"><a href="https://github.com/trilinos/Trilinos/issues/6540">#6540</a></td>
</tr>

<tr>
<td align="left">eclipse</td>
<td align="left"><a href="https://testing.sandia.gov/cdash/index.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=2&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=site&compare2=61&value2=eclipse">Trilinos-atdm-cts1empire-intel-18.0.2_&shy;openmpi-4.0.1_&shy;openmp_&shy;static_&shy;opt</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/27154158">SEACASIoss_&shy;structured_&shy;cgns_&shy;roundtrip_&shy;with_&shy;flush</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/27154158"><font color="green">Passed</font></a></td>
<td align="left">Completed</td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_with_flush&field3=site&compare3=61&value3=eclipse"><font color="green">29</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_with_flush&field3=site&compare3=61&value3=eclipse"><font color="red">0</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-07-25&end=2020-08-23&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_with_flush&field3=site&compare3=61&value3=eclipse"><font color="green">29</font></a></td>
<td align="right"><a href="https://github.com/trilinos/Trilinos/issues/6540">#6540</a></td>
</tr>

</table>


</details>

</body>

</html>

This is an automated comment generated by Grover. Each week, Grover collates and reports data from CDash in an automated way to make it easier for developers to stay on top of their issues. Grover saw that there are tests being tracked on CDash that are associated with this open issue. If you have a question, please reach out to Ross. I'm just a cat.

**Comment by grover-trilinos** [comment]
<html>

<body>

<h2>Test results for issue #6540 as of 2020-08-30</h2>

<p>
<font color="green">Tests with issue trackers Passed: twip=6</font><br>
</p>

<details>

<summary><b>Detailed test results:</b> (click to expand)</b></summary>

<h3><font color="green">Tests with issue trackers Passed: twip=6</font></h3>
<table style="width:100%" boarder="1">

<tr>
<th>Site</th>
<th>Build Name</th>
<th>Test Name</th>
<th>Status</th>
<th>Details</th>
<th>Consec&shy;utive Pass Days</th>
<th>Non-pass Last 30 Days</th>
<th>Pass Last 30 Days</th>
<th>Issue Tracker</th>
</tr>

<tr>
<td align="left">eclipse</td>
<td align="left"><a href="https://testing.sandia.gov/cdash/index.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=2&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=site&compare2=61&value2=eclipse">Trilinos-atdm-cts1empire-intel-18.0.2_&shy;openmpi-4.0.1_&shy;openmp_&shy;static_&shy;dbg</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/28621599">SEACASIoss_&shy;structured_&shy;cgns_&shy;append</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/28621599"><font color="green">Passed</font></a></td>
<td align="left">Completed</td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_append&field3=site&compare3=61&value3=eclipse"><font color="green">27</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_append&field3=site&compare3=61&value3=eclipse"><font color="red">0</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_append&field3=site&compare3=61&value3=eclipse"><font color="green">27</font></a></td>
<td align="right"><a href="https://github.com/trilinos/Trilinos/issues/6540">#6540</a></td>
</tr>

<tr>
<td align="left">eclipse</td>
<td align="left"><a href="https://testing.sandia.gov/cdash/index.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=2&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=site&compare2=61&value2=eclipse">Trilinos-atdm-cts1empire-intel-18.0.2_&shy;openmpi-4.0.1_&shy;openmp_&shy;static_&shy;opt</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/28613647">SEACASIoss_&shy;structured_&shy;cgns_&shy;append</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/28613647"><font color="green">Passed</font></a></td>
<td align="left">Completed</td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_append&field3=site&compare3=61&value3=eclipse"><font color="green">28</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_append&field3=site&compare3=61&value3=eclipse"><font color="red">0</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_append&field3=site&compare3=61&value3=eclipse"><font color="green">28</font></a></td>
<td align="right"><a href="https://github.com/trilinos/Trilinos/issues/6540">#6540</a></td>
</tr>

<tr>
<td align="left">eclipse</td>
<td align="left"><a href="https://testing.sandia.gov/cdash/index.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=2&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=site&compare2=61&value2=eclipse">Trilinos-atdm-cts1empire-intel-18.0.2_&shy;openmpi-4.0.1_&shy;openmp_&shy;static_&shy;dbg</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/28621626">SEACASIoss_&shy;structured_&shy;cgns_&shy;roundtrip_&shy;reordered_&shy;zones_&shy;fields</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/28621626"><font color="green">Passed</font></a></td>
<td align="left">Completed</td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_reordered_zones_fields&field3=site&compare3=61&value3=eclipse"><font color="green">27</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_reordered_zones_fields&field3=site&compare3=61&value3=eclipse"><font color="red">0</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_reordered_zones_fields&field3=site&compare3=61&value3=eclipse"><font color="green">27</font></a></td>
<td align="right"><a href="https://github.com/trilinos/Trilinos/issues/6540">#6540</a></td>
</tr>

<tr>
<td align="left">eclipse</td>
<td align="left"><a href="https://testing.sandia.gov/cdash/index.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=2&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=site&compare2=61&value2=eclipse">Trilinos-atdm-cts1empire-intel-18.0.2_&shy;openmpi-4.0.1_&shy;openmp_&shy;static_&shy;opt</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/28613674">SEACASIoss_&shy;structured_&shy;cgns_&shy;roundtrip_&shy;reordered_&shy;zones_&shy;fields</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/28613674"><font color="green">Passed</font></a></td>
<td align="left">Completed</td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_reordered_zones_fields&field3=site&compare3=61&value3=eclipse"><font color="green">28</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_reordered_zones_fields&field3=site&compare3=61&value3=eclipse"><font color="red">0</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_reordered_zones_fields&field3=site&compare3=61&value3=eclipse"><font color="green">28</font></a></td>
<td align="right"><a href="https://github.com/trilinos/Trilinos/issues/6540">#6540</a></td>
</tr>

<tr>
<td align="left">eclipse</td>
<td align="left"><a href="https://testing.sandia.gov/cdash/index.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=2&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=site&compare2=61&value2=eclipse">Trilinos-atdm-cts1empire-intel-18.0.2_&shy;openmpi-4.0.1_&shy;openmp_&shy;static_&shy;dbg</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/28621642">SEACASIoss_&shy;structured_&shy;cgns_&shy;roundtrip_&shy;with_&shy;flush</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/28621642"><font color="green">Passed</font></a></td>
<td align="left">Completed</td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_with_flush&field3=site&compare3=61&value3=eclipse"><font color="green">27</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_with_flush&field3=site&compare3=61&value3=eclipse"><font color="red">0</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_with_flush&field3=site&compare3=61&value3=eclipse"><font color="green">27</font></a></td>
<td align="right"><a href="https://github.com/trilinos/Trilinos/issues/6540">#6540</a></td>
</tr>

<tr>
<td align="left">eclipse</td>
<td align="left"><a href="https://testing.sandia.gov/cdash/index.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=2&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=site&compare2=61&value2=eclipse">Trilinos-atdm-cts1empire-intel-18.0.2_&shy;openmpi-4.0.1_&shy;openmp_&shy;static_&shy;opt</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/28613678">SEACASIoss_&shy;structured_&shy;cgns_&shy;roundtrip_&shy;with_&shy;flush</a></td>
<td align="left"><a href="https://testing.sandia.gov/cdash/test/28613678"><font color="green">Passed</font></a></td>
<td align="left">Completed</td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_with_flush&field3=site&compare3=61&value3=eclipse"><font color="green">28</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_with_flush&field3=site&compare3=61&value3=eclipse"><font color="red">0</font></a></td>
<td align="right"><a href="https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&begin=2020-08-01&end=2020-08-30&filtercombine=and&filtercombine=&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=61&value1=Trilinos-atdm-cts1empire-intel-18.0.2_openmpi-4.0.1_openmp_static_opt&field2=testname&compare2=61&value2=SEACASIoss_structured_cgns_roundtrip_with_flush&field3=site&compare3=61&value3=eclipse"><font color="green">28</font></a></td>
<td align="right"><a href="https://github.com/trilinos/Trilinos/issues/6540">#6540</a></td>
</tr>

</table>


</details>

</body>

</html>

This is an automated comment generated by Grover. Each week, Grover collates and reports data from CDash in an automated way to make it easier for developers to stay on top of their issues. Grover saw that there are tests being tracked on CDash that are associated with this open issue. If you have a question, please reach out to Ross. I'm just a cat.

**Comment by gsjaardema** [comment]
It looks like all tests are passing.
